config/main.py

```diff
@@ -645,7 +645,7 @@ def _get_disabled_services_list(config_db):
 
     feature_table = config_db.get_table('FEATURE')
     if feature_table is not None:
-        for feature_name in feature_table.keys():
+        for feature_name in list(feature_table.keys()):
```

Comment:
>list [](start = 28, length = 4)

The same

Reply:
This change was made by the `2to3` tool (see [here](https://www.python.org/dev/peps/pep-0469/#migrating-directly-to-python-3)):

> Note that in many cases d.keys() can be replaced by just d, but the 2to3 migration tool doesn't attempt that replacement.

Plus, keeping `.keys()` makes it clear what the intention is.